plugins/qcheck-stm/test/dune

```diff
@@ -98,6 +105,26 @@
    (diff hashtbl_errors.expected hashtbl_errors)
    (diff hashtbl_stm_tests.expected.ml hashtbl_stm_tests.ml))))
 
+(rule
```

Comment:
It is strange to see this inserted in the middle of the `hashtbl`-related stanzas.